client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -138,8 +138,11 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
   const customURLValidation = (schema: StringSchema) => {
     const gitUrlRegex =
       /(?:git|ssh|https?|git@[-\w.]+):(\/\/)?(.*?)(\.git)(\/?|\#[-\d\w._]+?)$/;
+
+    const standardUrlRegex = new RegExp(standardURLRegex);
+
     const containsURL = (string: string) =>
-      gitUrlRegex.test(string) || standardURLRegex.test(string);
+      gitUrlRegex.test(string) || standardUrlRegex.test(string);
```

Comment:
Slightly confusing naming here with the only difference between standardUrlRegex and standardURLRegex being the case. Just a suggestion.

Reply:
@ibolton336,
I haven't been to the bottom of it but I wish we could avoid creating a RegExp from a RegExp.
Naming is hard. 
What about "localStandardURLRegex" ?

Reply:
Sounds like a move in the right direction. I liked the pattern of wrapping the regex test in a function and just calling that wherever we needed it instead of having to use .test every time.